packages/appstore/src/components/wallet-header/wallet-header.tsx

```diff
@@ -1,73 +1,71 @@
 import React from 'react';
 import { Icon } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
 import classNames from 'classnames';
 import WalletCurrencyCard from './wallet-currency-card';
 import WalletHeaderButtons from './wallet-header-buttons';
 import WalletHeaderTitle from './wallet-header-title';
 import WalletHeaderBalance from './wallet-header-balance';
-import { TAccountCategory, TAccountStatus, TWalletShortcode, TWalletCurrency } from 'Types';
+import { TWalletAccount } from 'Types';
 import { getWalletHeaderButtons } from 'Constants/utils';
 import './wallet-header.scss';
 
 type TWalletHeader = {
-    account_type: TAccountCategory;
-    shortcode?: TWalletShortcode;
-    currency?: TWalletCurrency;
-    balance?: string;
-    account_status?: TAccountStatus;
-    is_open_wallet: boolean;
-    setIsOpen: (is_open: boolean) => void;
+    wallet_account: TWalletAccount;
 };
 
-const WalletHeader = React.memo(
-    ({
-        account_status = '',
-        balance = '0.00',
-        currency = 'USD',
-        shortcode = 'svg',
-        account_type = 'real',
-        is_open_wallet,
-        setIsOpen,
-    }: TWalletHeader) => {
-        const is_demo = account_type === 'demo';
+const WalletHeader = observer(({ wallet_account }: TWalletHeader) => {
+    const { client, traders_hub } = useStore();
+    const { switchAccount, loginid } = client;
+    const is_active = loginid === wallet_account.loginid;
```

Comment:
Just a question, Can't we use `useActiveWallet` here?

Reply:
Will use is_selected instead 😸